tsconfig.json

```diff
@@ -16,6 +16,10 @@
       "Assets/*": ["./Assets/*"]
     }
   },
-  "include": ["typings/*.d.ts", "src/**/*.ts", "src/**/*.tsx"],
-  "exclude": ["node_modules", "dist", "data", "externals", "danger"]
+  "include": [
+    "node_modules/styled-components/typings/styled-components.d.ts",
```

Comment:
We actually don't need that line anymore but we can keep it for now. I'm still working out some kinks to swap babel with tsc in this PR. https://github.com/artsy/reaction/pull/715. I'll update it later today.

Reply:
@l2succes I'll remove just to keep things clear